test/unit/mdc-top-app-bar/mdc-top-app-bar.test.js

```diff
@@ -57,16 +57,41 @@ function setupTest() {
   const fixture = getFixture();
   const root = fixture.querySelector('.mdc-top-app-bar');
   const adjust = fixture.querySelector('.mdc-top-app-bar-fixed-adjust');
-  const component = new MDCTopAppBar(root);
-  return {root, adjust, component};
+  const icon = root.querySelector('.mdc-top-app-bar__menu-icon');
+  const component = new MDCTopAppBar(root, undefined, (el) => new FakeRipple(el));
+
+  return {root, adjust, component, icon};
+}
+
+class FakeRipple {
+  constructor(root) {
+    this.root = root;
+    this.destroy = td.func('.destroy');
+    this.unbounded = null;
+  }
 }
 
 suite('MDCTopAppBar');
 
-test('attachTo initializes and returns an MDCTopAppBar instance', () => {
+test('initialize and returns an MDCTopAppBar instance', () => {
   assert.isTrue(MDCTopAppBar.attachTo(getFixture()) instanceof MDCTopAppBar);
 });
 
+test('constructor instantiates icon ripples', () => {
+  const {root, component} = setupTest();
+  const icons = root.querySelectorAll('.mdc-top-app-bar__icon').length + 1;
```

Comment:
Can this reuse the two `*_ICON_SELECTOR` constants together in one selector and forego the +1?